mockgcp/mockcompute/networksv1.go

```diff
@@ -68,6 +68,12 @@ func (s *NetworksV1) Insert(ctx context.Context, req *pb.InsertNetworkRequest) (
 	obj.SelfLink = PtrTo(buildComputeSelfLink(ctx, name.String()))
 	obj.SelfLinkWithId = PtrTo(buildComputeSelfLink(ctx, fmt.Sprintf("projects/%s/global/networks/%d", name.Project.ID, id)))
 	obj.Kind = PtrTo("compute#network")
+	obj.NetworkFirewallPolicyEnforcementOrder = PtrTo("AFTER_CLASSIC_FIREWALL")
```

Comment:
Perhaps we should only assign the default value when the request doesn't explicitly specify one. But we can fix later when we actually test this field.

https://github.com/GoogleCloudPlatform/k8s-config-connector/blob/5d311a1c4e1429efd7667fe43cc041fb70a818d5/tests/apichecks/testdata/exceptions/missingfields.txt#L768